docs/guide.rst

```diff
@@ -2,4 +2,76 @@
 User Guide
 **********
 
-This user guide provides a walkthrough of the major features in the ``hvpy`` package.
+Welcome to ``hvpy``!
+
+This guide provides a walkthrough of the major features in the ``hvpy`` package.
+``hvpy`` is a python wrapper for the `Helioviewer API <https://api.helioviewer.org/docs/v2/>`_.
+This is the same API that powers `helioviewer.org <https://helioviewer.org>`_.
```

Comment:
```suggestion
``hvpy`` is a python wrapper for the `Helioviewer API <https://api.helioviewer.org/docs/v2/>`__.
This is the same API that powers `helioviewer.org <https://helioviewer.org>`__.
```
Double __ are used for inline urls to avoid potential duplication errors in sphinx.

Reply:
Updated with double __